docs/computing/connecting/index.md

```diff
@@ -0,0 +1,164 @@
+# Connecting to CSC supercomputers
+
+There are two main ways of connecting to CSC supercomputers. The traditional way to
+work on a supercomputer is [using an SSH client](#using-an-ssh-client). We
+also offer a [web interface](#using-the-web-interface) to our systems, which
+enables running both graphical applications and command-line shells.
+
+For instructions on connecting to the LUMI supercomputer, please see the
+[Get Started page in the LUMI user guide](https://docs.lumi-supercomputer.eu/firststeps/getstarted/).
+
+!!! note "Login node usage policy"
+
+    When you connect to a supercomputer using an SSH client or the
+    [*Login node shell*](../webinterface/shell.md) app, you are directed to a
+    login node. Login nodes are **not meant for long or heavy processing**.
+    The accepted uses for login nodes are defined in our
+    [login node usage policy](../usage-policy.md#login-nodes).
+
+## Using the web interface
+
+The [supercomputer web interface](../webinterface/index.md) is a good platform
+for using most graphical applications, such as Jupyter notebooks and RStudio, on
+Puhti and Mahti. Additionally, you can launch a
+[remote desktop](../webinterface/desktop.md) for running various programs
+that feature graphical user interfaces. It is also possible to open a
+persistent shell on a login or compute node, which will keep running even if
+you close your browser or lose your internet connection. The command-line
```

Comment:
Only compute node shell is persistent, not login node shell.

Reply:
4ca07540 fixes this and improves general legibility of the section